### ISSUE TITLE
Steam Deck UI issue

### ISSUE THREAD

**Issue by diannetea** [Annosz/UIInfoSuite2#307]
On the steam deck it is impossible to get to the options tab with the controls. I tried looking for another way to configure it (file) bit I didn't see one.

Could it be possible for the options to show in the Generic Mod Config Menu if the tab is disabled?  Thanks.

**Comment by biomagnet** [comment]
Yeah, I agree with this. Can't use the shoulder buttons to get to the menu option, and my small hands don't like just reaching for the touch screen when controls would be easier.

**Comment by tqdv** [comment]
Hi @diannetea and @biomagnet,
I just released version 2.2.10 which implements gamepad controls for the gamemenu.
If you could test it and tell me if it works, I would I greatly appreciate it.
Download link is here: https://github.com/Annosz/UIInfoSuite2/releases/download/2.2.10/UIInfoSuite2-v2.2.10.zip
